lucille-core/src/test/resources/MatchQueryTest/blank_matched_queries.conf

```diff
@@ -0,0 +1,21 @@
+# This is a configuration file managed by https://github.com/lightbend/config
```

Comment:
please omit these two boilerplate comment lines here and in the other conf files for MatchQueryTest. (We have lots of .conf files in the codebase and it doesn't make sense include this comment everywhere.)